core/amber/src/main/scala/edu/uci/ics/texera/web/resource/WorkflowWebsocketResource.scala

```diff
@@ -50,14 +57,18 @@ class WorkflowWebsocketResource {
   def myOnOpen(session: Session, config: EndpointConfig): Unit = {
     WorkflowWebsocketResource.sessionMap.update(
       session.getId,
-      (session, config.getUserProperties.get("httpSession").asInstanceOf[HttpSession])
+      (
+        session,
+        config.getUserProperties.get("httpSession").asInstanceOf[HttpSession]
+      )
```

Comment:
I think your scala formatter has different settings, can you revert those format changes (and change your formatter config)?